libraries/redcore/database/driver.php

```diff
@@ -113,7 +111,7 @@ public static function getInstance($options = array())
 				// Create our new JDatabase connector based on the options given.
 				try
 				{
-					$instance = new $class($options);
+					$instance = $class::getInstance($options);
```

Comment:
getInstance will create a new one actually
that was the Travis error in fact, because those drivers's constructors are protected so your line was not working